### PR TITLE
OXT-168: add a brief README file. With a diagram!

### PR DESCRIPTION
**Pull request by dickon**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+Note that client.py is a short running process which maintains no state. It runs once
+to completion and then exits. Normally client.py is started by launcher.py which handles
+getting the configuring information for client.py and running it regularly, and handling exits.
+
+## Dependencies
+
+sync-client requires:
+
+* a synchronizer server to get target state and VHDs from (see https://github.com/openxt/sync-server/). In
+turn the synchronizer server will require:
```
Comment (pearsonk) [inline README.md:24]: Probably shouldn't cite what synchronizer requires on the server side. That should be in sync-server which is already referenced here

File: README.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+** sync-database, a schema and stored procedures
+** sync-cli, commnad line tools to access that database
+** sync-wui, an optional web user interface for administrators
+* icbinn (https://github.com/openxt/icbinn) to provide specific filesystem access to dom0 to write VHD files
+* Python 2, (http://python.org)
+* dbus-python (https://pypi.python.org/pypi/pydbus/0.2)
```
Comment (pearsonk) [inline README.md]: Don't forget the ever-lovely V4V requirement for wrapping the D-Bus

**Comment by pearsonk** [comment]
In the diagram, may want to put the filesystem for VHDs is not local (Dom0 access via icbinn).

**Comment by pearsonk** [comment]
Ignore previous V4V comments, seems that the V4V portion is handled elsewhere by a quick scan of the code in this repo

**Comment by flihp** [comment]
I'm inclined to merge this as-is unless there are objections. @pearsonk : I agree that a division between the sync client VM and dom0 in the diagram would be useful but I don't see a need to hold this up (better than no documentation at all). We can always make these better down the line.

**Pull request merged**